packages/db/mocks/comment.mock.ts

```diff
@@ -5,9 +5,14 @@ import { trashId } from '../seed';
  *
  * @returns Creates a mock user.
  */
-export default function CommentMock(parentId?: number): Prisma.CommentCreateManyInput {
+export default function CommentMock(
+  commentNumber: number,
+  parentId?: number,
+): Prisma.CommentCreateManyInput {
   return {
-    text: 'here is a comment',
+    text: `here is a comment ${
+      !parentId ? commentNumber : `reply ${commentNumber} to parent ${parentId}`
+    }`,
     userId: trashId,
```

Comment:
for a random user i think in the seed script we pull all the user IDs, so you could use the `faker.helpers.arrayElement` to choose a random userId instead of hard-coding it to trash's

Reply:
yeah I'll do this in a separate PR lol